soco/core.py

```diff
@@ -1811,6 +1813,23 @@ def get_tracks_for_album(self, artist, album, full_album_art_uri=False):
         result._metadata['search_type'] = 'tracks_for_album'
         return result
 
+    @property
+    def library_updating(self):
+        """True if the music library is in the process of being updated
+
+        :returns: True if the music library is in the process of being updated
+        :rtype: bool
+        """
+        result = self.contentDirectory.GetShareIndexInProgress()
+        return result['IsIndexing'] != '0'
+
+    def start_library_update(self):
+        """Start an update of the music library."""
+        return self.contentDirectory.RefreshShareIndex([
+            ('AlbumArtistDisplayOption', ''),
```

Comment:
Perhaps expose this as a parameter, with a default value?  Does this have something to do with the "compilation Albums" settings from the official controller? Any idea what values it can take?